components/chat/conversations/Conversation.tsx

```diff
@@ -5,19 +5,17 @@ import Colors from '../../../theme/colors';
 
 interface ConversationProps {
   data: TConversation;
-  lastMessage: LastMessage;
 }
 
-const Conversation = ({ data, lastMessage }: ConversationProps) => {
-  const { msg } = lastMessage;
+const Conversation = ({ data }: ConversationProps) => {
   return (
     <View style={styles.cardContainer}>
       <View style={styles.conversationRowContainer}>
         <Avatar src={data.bot.profileImage} width={40} height={40} />
         <View style={styles.conversationInfoContainer}>
           <Text style={styles.conversationTitle}>{data.title}</Text>
           <Text numberOfLines={1} ellipsizeMode="tail" style={styles.conversationLastMessage}>
-            {msg}
+            {data.lastMessage}
```

Comment:
Can we truncate this so for long messages it looks like: "text message messa..."

Reply:
we already do it